seed/rbnf/nci.xml

```diff
@@ -11,7 +11,7 @@ For terms of use, see http://www.unicode.org/copyright.html
         <language type="nci"/>
     </identity>
     <rbnf>
-        <rulesetGrouping type="SpelloutRules" draft="unconfirmed">
```

Comment:
The better choice is to add rulesetGrouping to DtdData.DRAFT_ON_NON_LEAF_ALLOWED

Reply:
Wherever the test message is surfacing, we should add a pointer to DtdData.DRAFT_ON_NON_LEAF_ALLOWED, but with a caution that it should only be expanded if we are certain that it is ok to have draft on a non-leaf (should be rare).

Reply:
> The better choice is to add rulesetGrouping to DtdData.DRAFT_ON_NON_LEAF_ALLOWED

Ok great thanks. I had assumed the behavior was intended, but will fix.

Reply:
see message below